admin/SWC_cover_letter.txt

```diff
@@ -0,0 +1,18 @@
+Dear SCF Mentoring Subcommittee,
+
+We are writing to express our interested to attend the two-day training course in December 2015 as one group composed of:
+
+Madeleine Bonsma (@mbonsma on GitHub),
+Lindsay Coome (@lcoome on GitHub),
+Lina Tran (@linamnt on Github),
+Sarah Meister (@thesarahmeister on GitHub),
+Elliott Sales de Andrade (@QuLogic on GitHub) and
+Ricardo Harripaul (@ricardoharripaul on GitHub).
+
+Madeleine Bonsma is the organizer of a scientific progamming peer learning group at the University of Toronto (University of Toronto Scientific Coders - uoftcoders.github.io) and is on the waitlist for instructor training. Lindsay Coome is a session leader in the University of Toronto Scientific Coders group and is on the waitlist for instructor training. Lina Tran is an organizer and member of the University of Toronto Scientific Coders group. Sarah Meister is a session leader in the Scientific Coders group. Ricardo Harripaul is a session leader in the Scientific Coders group. Elliott Sales de Andrade is an organizer and member of the University of Toronto Scientific Coders group.
```

Comment:
for Lina Tran: "...she also attended a Software Carpentry workshop at Sickkids Hospital in 2015."